src/Features/Core/Portable/QuickInfo/CommonSemanticQuickInfoProvider.cs

```diff
@@ -349,7 +354,15 @@ private ImmutableArray<ISymbol> GetSymbolsFromToken(SyntaxToken token, Workspace
         {
             if (GetBindableNodeForTokenIndicatingLambda(token, out var lambdaSyntax))
             {
-                return ImmutableArray.Create(semanticModel.GetSymbolInfo(lambdaSyntax, cancellationToken).Symbol);
```

Comment:
If I wanted to perfectly preserve the old code, I should have had this function return `ImmutableArray<ISymbol?>`, which worked because the IsOk function handled null inputs. That was crazy, so I just fixed it instead.